mobius/src/algorithms/grid_ifs.rs

```diff
@@ -74,6 +74,14 @@ impl<G: Group> GridIFS<G> {
             .map(|(_, xform)| primitive.transform(xform))
             .collect()
     }
+
+    pub fn flat_apply<T>(&self, primitive: &T) -> T
```

Comment:
Document these methods to explain that the purpose is for combining collections of primitives into a single collection, which is handy when rendering.

`apply()` might still be handy if the point is to do something other than rendering (e.g. do some other computation for each tile